gapis/resolve/command_tree.go

```diff
@@ -342,8 +342,9 @@ func (r *CommandTreeResolvable) Resolve(ctx context.Context) (interface{}, error
 	for k, v := range snc.SubcommandGroups {
 		r := out.root.AddRoot([]uint64{uint64(k)})
 		// subcommands are added before nested SubCmdRoots.
-		sort.SliceStable(v, func(i, j int) bool { return len(v[i]) < len(v[j]) })
-		for _, x := range v {
+		cv := append([]api.SubCmdIdx{}, v...)
+		sort.SliceStable(cv, func(i, j int) bool { return len(cv[i]) < len(cv[j]) })
```

Comment:
Why don't we just insist on having the provider of the `SubcommandGroups` sort things?